retroshare-gui/src/util/HandleRichText.cpp

```diff
@@ -39,11 +39,14 @@
 #include "util/rstime.h"
 
 #ifdef USE_CMARK
-//Include for CMark
-// This needs to be fixed: use system library if available, etc.
-#include <gui/../../../supportlibs/cmark/src/cmark.h>
-#include <gui/../../../supportlibs/cmark/src/node.h>
-#endif
+  //Include for CMark
+  #ifdef HAS_CMARK
+    #include <cmark.h>
+  #else
+    #include <gui/../../../supportlibs/cmark/src/cmark.h>
```

Comment:
This ugly include can be avoided by adding cmark headers path to INCLUDEPATH

Reply:
it can also be avoided by not "hardcoding" headers into RS.